### PR TITLE
Update deps

### PR DESCRIPTION
**Pull request by hzoo**
recast isn't used since it's forked inline in src

### CODE REVIEW

**Comment by jlongster** [comment]
Oh right, thanks! You need to remove the recast import in `tests_config/run_spec.js`. We used to use it in there but not anymore.

**Comment by hzoo** [comment]
Oops missed that one! (only checked source)

File: tests_config/run_spec.js [hunk 1]
```diff
@@ -1,7 +1,6 @@
 "use strict";
 const fs = require('fs');
 const jscodefmt = require("../");
```
Comment (vramana) [inline tests_config/run_spec.js]: Probably we should change this to `prettier` as well. 😄

Comment (jlongster) [inline tests_config/run_spec.js]: I'll do that in a follow-up!

**Pull request merged**